CONTRIBUTING.md

```diff
@@ -15,14 +15,62 @@ Thank you for your interest in contributing to this project! Your help is greatl
 
 ## How to Contribute
 
-1. **Fork the repository**: Start by forking the repository to your own GitHub account.
-2. **Create a branch**: Make sure to create a feature branch (`git checkout -b feature-name`) before starting any development.
-3. **Make your changes**: Implement your feature, bugfix, or documentation update in your forked repository.
-4. **Write tests**: Ensure that new features or bugfixes are covered by appropriate tests.
-5. **Commit your changes**: Write clear, descriptive commit messages. Follow the project's commit message format.
-6. **Push your branch**: Push your feature branch to your forked repository.
-7. **Create a pull request (PR)**: Open a pull request against the main branch of the original repository.
+### 1. Feature Requests
+We love hearing your ideas! If you have a feature request, please create a new issue and describe the feature you would like to see, including its potential benefits.
 
+### 2. Contributing Code
+
+If you'd like to contribute code, please follow these steps:
+
+#### Fork the Repository
+
+1. Click the "Fork" button in the upper right corner of the repository.
+
+2. Clone your forked repository to your local machine:
+```bash
+   git clone https://github.com/yourusername/backend-generator-cli.git
```

Comment:
please update backend-generator-cli to the repo name

Reply:
Yaa sure